### DISCUSSION TITLE
Modelling heterogeneity in starting time

### DISCUSSION THREAD

**Discussion by dryoutao** [nlmixrdevelopment/nlmixr#583] [Q&A]
Hi,

I am trying to model disease transmission via contact, where naive animals were caged with an infected animal carrying a virus that is communicable via airborne route.

The point in time for each naive animal to be infected is different: infection starts somewhere between time 0 and 72h.

Is it possible to incorporate a random effect, say eta.t0, to handle this in a ODE model such as the following?

a(0) = 1
v(0) = v0
d/dt(a) = -beta * a * v
d/dt(v) = gamma * a * v - delta * v

(This model specifies initial time at time 0, not at eta.t0)

Best,
Tao

**Comment by mattfidler** [comment]
Maybe:

```r
d/dt(a) = -beta * a * v
d/dt(v) = gamma * a * v - delta * v
alag(a) = exp(eta.t0)
alag(v) = exp(eta.t0)
f(v) = v0
```

with a dose of 1 to `a` and `v` compartments
